lib/simp/cli/config/items/data/svckill_mode.rb

```diff
@@ -0,0 +1,64 @@
+require File.expand_path( '../item', File.dirname(__FILE__) )
+
+module Simp; end
+class Simp::Cli; end
+module Simp::Cli::Config
+  class Item::SvckillMode < Item
+    def initialize
+      super
+      @key         = 'svckill::mode'
+      @description = %Q{
```

Comment:
The description has an extra line after the === svckill::mode === declaration, which does not follow the format in the rest of simp config.